autoload/ale/fix/registry.vim

```diff
@@ -375,11 +390,6 @@ let s:default_registry = {
 \       'suggested_filetypes': ['html', 'htmldjango'],
 \       'description': 'Fix HTML files with html-beautify.',
 \   },
-\   'dhall': {
```

Comment:
Add an alias to `dhall-format` for this, so anyone who is currently using the fixer under this name can keep using it.